google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineFacet.java

```diff
@@ -38,6 +40,8 @@ public AppEngineFacet(@NotNull FacetType facetType,
       @NotNull String name,
       @NotNull AppEngineFacetConfiguration configuration) {
     super(facetType, module, name, configuration, null);
+
+    UsageTrackerProvider.getInstance().trackEvent(GctTracking.APP_ENGINE_CREATE_FACET).ping();
```

Comment:
is the facet added to a module in code that we manage?  I'm thinking AE facets added to modules is closer to the signal we want.